<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <arg name="params_file"/>

  <node name="slam_toolbox"
        pkg="slam_toolbox"
        type="async_slam_toolbox_node"
        output="screen">
    <rosparam file="$(arg params_file)" />
  </node>

</launch>
